app/src/main/java/com/keylesspalace/tusky/network/InstanceSwitchAuthInterceptor.java

```diff
@@ -49,8 +49,13 @@ public Response intercept(@NonNull Chain chain) throws IOException {
 
             Request.Builder builder = originalRequest.newBuilder();
 
+            String authorizationHeader = originalRequest.header(MastodonApi.TOKEN_HEADER);
```

Comment:
As we expect for #517 it is going to be a common case to use another account and because interceptor already knows about accounts maybe it's a good idea to pass account ID instead of token header for every request? We should think about priority it needs to have but I think it should be lower than instance but should override active account